### PR TITLE
add basic flow to buttons & icons

### PR DESCRIPTION
**Pull request by ortalsMoveo**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import './App.css';
 import MainPage from './fetchers/MainPage/MainPage';
-
 import WelcomePage from './fetchers/WelcomePage/WelcomPage';
-const device = 'Desktop';
+
+const device = 'Tablet';
```
Comment (amitnirG) [inline src/App.tsx]: why you still need device?

File: src/components/Card/Card.tsx [hunk 2]
```diff
@@ -50,7 +50,8 @@ const Card = ({imagePath, tags, cardData}: CardProp) => {
                 <ButtonContainer>
                     <Button 
                         buttonText={primaryButton.buttonText}
-                        icon={primaryButton.icon} 
+                        icon={primaryButton.icon}
+                        onClickFunc={() => console.log(cardData.title)}
```
Comment (amitnirG) [inline src/components/Card/Card.tsx:54]: this needs to be OnClick not OnClickFunc

File: src/App.tsx (hunk 1 above)
Comment (ortalsMoveo) [reply src/App.tsx]: var deleted.

File: src/components/Icon/Icon.tsx [hunk 3]
```diff
@@ -1,10 +1,14 @@
+import React from "react";
```
Comment (amitnirG) [inline src/components/Icon/Icon.tsx:1]: looks like this component doesn't do anything but returning an img.
you can remove it and use normal img instead unless you are planning to do something with it.

File: src/components/Tablet&Mobile/FilterComponent/Filter.stories.tsx [hunk 4]
```diff
@@ -9,11 +9,13 @@ export default {
...
-const Template: Story = (args) => <Filter {...args} />;
+const Template: Story<FilterProps> = (args) => <Filter {...args} />;
+
 
 // Here we define variants
 export const Primary = Template.bind({});
 Primary.args = {
-  sortbyIcon: downArrow,
-  icon: icon
+  // sortbyIcon: downArrow,
```
Comment (amitnirG) [inline src/components/Tablet&Mobile/FilterComponent/Filter.stories.tsx:18]: remove comments

File: src/components/Tablet&Mobile/FilterTablet/FilterTablet.tsx [hunk 5]
```diff
@@ -49,6 +49,7 @@ const FilterTablet = ({title, icon,subFilter, list, button}: FilterProps) => {
             <ButtonPosition>
                 <Button 
                     buttonText={button.buttonText}
+                    onClickFunc={() => console.log(button.buttonText)}
```
Comment (amitnirG) [inline src/components/Tablet&Mobile/FilterTablet/FilterTablet.tsx:52]: same here about onClickFunc

File: src/fetchers/MainPage/MainPage.tsx [hunk 6]
```diff
@@ -26,13 +26,20 @@ const recentSearches = ["crypto", "soccer", "soccer"];
...
+
   const [filterTabletOn, setFilterTabletOn] = useState(true);
   const DesktopSize = useWindowDimensions();
   
   //fetch data from server
   // const [dataCards, setDataCards] = useState([])
   // const [dataGraphs, setDataGraphs] = useState([]);
 
+  const closeSidebar = () =>{
+    if(filterTabletOn){
```
Comment (amitnirG) [inline src/fetchers/MainPage/MainPage.tsx:38]: generally the word mobile is better than tablet. I recommend to change it across the project.
change filterTabletOn => filterMobileOpen. its more clear

File: src/fetchers/WelcomePage/WelcomPage.tsx [hunk 7]
```diff
@@ -17,6 +21,7 @@ const WelcomePage = () => {
                         buttonText="continue"
                         icon={true}
                         fullWidth={true}
+                        onClickFunc={onClickHandler}
```
Comment (amitnirG) [inline src/fetchers/WelcomePage/WelcomPage.tsx:24]: same here

**Pull request merged**